.gitignore

```diff
@@ -0,0 +1,2 @@
+*.xcuserstate
+xcuserdata/
```

Comment:
Please use the [standard gitignore](https://github.com/apple/swift-package-manager/blob/41b97c87dfd37dc263abc4975a65b7d5042e00be/Sources/Workspace/InitPackage.swift#L257-L263) that SPM provides.